src/containers/Header/Header.test.jsx

```diff
@@ -50,6 +53,54 @@ describe('<Header />', () => {
     store = mockStore({
       portalConfiguration: {},
       authentication: {},
+      userProfile: {},
+    });
+    tree = renderer
+      .create((
+        <HeaderWrapper store={store} />
+      ))
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders external profile image correctly', () => {
+    store = mockStore({
+      portalConfiguration: {},
+      authentication: {
+        email: 'staff@example.com',
+        username: 'staff',
+      },
+      userProfile: {
+        profile: {
+          profile_image: {
+            has_image: true,
+            image_url_medium: 'https://raw.githubusercontent.com/edx/edx-portal/master/src/images/edx-logo.png',
```

Comment:
I didn't mention it as its a) in a test and b) I don't think it will actually fetch the image, would just render the snapshot with a reference to it so if it was no longer there the test would pass. But it might not hurt to make it obvious its a dummy url like `https://path-to-profile-image`

Reply:
That's true it wouldn't actually be fetching the image in the snapshot, in which case a dummy URL would work. 

But, if we decide to eventually use visual regression testing/diffing, it would be fetched so we might as well import and use the actual `edx-logo.png` image here.